.github/workflows/ci.yml

```
@@ -936,6 +938,7 @@ jobs:
             - suite-iceberg
             - suite-hudi
             - suite-ignite
+            - suite-star-rocks
```

nit: for consistency

```suggestion
            - suite-starrocks
```